modules/nf-core/tcoffee/align/main.nf

```diff
@@ -0,0 +1,47 @@
+process TCOFFEE_ALIGN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::t-coffee=13.45.0.4846264"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/t-coffee:13.45.0.4846264--hc57179f_5':
+        'biocontainers/t-coffee:13.45.0.4846264--hc57179f_5' }"
+
+    input:
+    tuple val(meta) ,  path(fasta)
+    tuple val(meta2),  path(tree)
+    tuple val(meta3),  path(template), path(accessory_informations)
```

Comment:
For what are you using these inputs?

Reply:
The first one is the standard one, so it is the input fasta file to be aligned.

The second one (optional) is a guide tree, some modes of this tool can run without it, but sometimes you want to force it t use a certain guidetree that you precomputed. A guidtree defines the order in which sequences are aligned within the tool.

The third one (also optional) allows passing accessory information to run special modes of tcoffee. For instance, one mode of tcoffee uses protein structural information, another one uses secondary structure information. In this case, you also need to add a template file (this is specified in the tcoffee documentation on how to build it - it is an advanced mode) - it basically informs tcoffee on which accessory info ( e.g. which pdb file) to use for which sequence.

Reply:
Let me know if this is clear or if i can should more info :)

Reply:
Thanks for the explanation! Can you make it so that these files are automatically used by the module when they are given? This way you don't have to use a config to use these files. You can see an example in [this module](https://github.com/nf-core/modules/blob/451df5258e0c03cb4e835bb02c22a9a31b44eb0d/modules/nf-core/gatk4/haplotypecaller/main.nf#L31-L32).

Reply:
Good point! Done :)